circleci/images/pgupgradetester/files/sbin/install-testdeps

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# make bash behave
+set -euxo pipefail
+IFS=$'\n\t'
+
+# this script needs root to install packages
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+OLD_PG_VERSION=10
+NEW_PG_VERSION=11
+
+# explicitly set user/group IDs
+groupadd -r circleci --gid=1729
+useradd -mg circleci --uid=1729 --shell=/bin/bash circleci
```

Comment:
guess `circleci` requires this. Is there any use for the `postgres` user we create in the image?